x/gamm/keeper/pool.go

```diff
@@ -354,3 +354,9 @@ func asCFMMPool(pool poolmanagertypes.PoolI) (types.CFMMPoolI, error) {
 	}
 	return cfmmPool, nil
 }
+
+// GetTradingPairTakerFee is a wrapper for poolmanager's GetTradingPairTakerFee, and is solely used
+// to get access to this method for use in sim_msgs.go for the GAMM module.
+func (k Keeper) GetTradingPairTakerFee(ctx sdk.Context, denom0, denom1 string) (sdk.Dec, error) {
+	return k.poolManager.GetTradingPairTakerFee(ctx, denom0, denom1)
+}
```

Comment:
The issue is, we define the keepers as local values and not global ones, so we cannot call them in the simulator despite them technically being available. This felt like the cleanest workaround (it feels more extreme to export the entire keeper, since we do not do this anywhere else)

Reply:
Makes sense I propose this as a long-term solution: https://github.com/osmosis-labs/osmosis/issues/6185